### DISCUSSION TITLE
Can I use github copilot in neovim?

### DISCUSSION THREAD

**Discussion by 4lexxe** [community/community#67014] [Programming Help] [answered]
### Body

se puede?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by hafiz-muhammad** [comment, marked as answer]
Yes you can, here is a Neovim plugin for GitHub Copilot: [https://github.com/github/copilot.vim](https://github.com/github/copilot.vim)
